app/uk/gov/hmrc/economiccrimelevyregistration/views/NotLiableView.scala.html

```diff
@@ -0,0 +1,45 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.economiccrimelevyregistration.utils.EclTaxYear
+
+@this(
+        layout: templates.Layout,
+        govukInsetText : GovukInsetText
+)
+
+@(feedbackUrl: String)(implicit request: Request[_], messages: Messages)
```

Comment:
you don't need to pass `feedbackUrl` as a parameter into the view, you can inject `AppConfig` into your twirl templates as part of the `@this(...`